src/views/product-downloads-view/index.tsx

```diff
@@ -114,7 +115,10 @@ const ProductDownloadsViewContent = ({
 					<meta name="robots" key="robots" content="noindex, nofollow" />
 				</HashiHead>
 			) : null}
-			<PageHeader isEnterpriseMode={isEnterpriseMode} />
+			<PageHeader
+				isEnterpriseMode={isEnterpriseMode}
+				productName={installName}
+			/>
```

Comment:
thought: we could simplify `PageHeader` and avoid the fallbacks if we pass in `currentProduct.name` here:
```suggestion
			<PageHeader
				isEnterpriseMode={isEnterpriseMode}
				productName={installName || currentProduct.name}
			/>
```